webroot/js/autocompletion.js

```diff
@@ -69,35 +69,27 @@ function suggestSelect(suggestionStr) {
     return false;
 }
 /**
- * transform the xml result into html content
+ * transform the json result into html content
  */
-function suggestShowResults(xmlDocResults) {
+function suggestShowResults(suggestions) {
     // we remove the old one
     removeSuggestList();
-    suggestions = xmlDocResults.getElementsByTagName('item');
 
     if (suggestions.length == 0) {
         return;
     }
-    suggestLength = suggestions.length;
+
+    isSuggestListActive = true;
 
     var ul = document.createElement("ul");
     $("#autocompletionDiv").append(ul);
-    isSuggestListActive = true;
-    for (var i in suggestions) {
-        // for some weird reason the last element in suggestion is the number
-        // of element in the array Oo
-        // and even a function oO wow we're leaving in a strange world .... 
-        if (!isNaN(parseInt(suggestions[i])) || $.isFunction(suggestions[i])) {
-            continue;
-        }
-        suggestion = suggestions[i].firstChild.data;
+    suggestions.forEach(function(suggestion, index) {
         var li = document.createElement("li");
-        li.innerHTML = "<a id='suggestItem" + i + "' onclick='suggestSelect(this.innerHTML)' style='color:black;'>"+
-            suggestion +
-        "</a>";
+        li.innerHTML = `<a id='suggestedItem${index}' onclick='suggestSelect("${suggestion.name}")' style='color:black';>
+        ${suggestion.name} (${suggestion.nbrOfSentences})
+        </a>`;
```

Comment:
Could you rewrite this part without using backticks? They are not supported by IE.

Reply:
Thank you for letting me know this compatibility issue, I've fixed it.